cmd/buildctl/build/util.go

```diff
@@ -31,3 +32,16 @@ func loadGithubEnv(cache client.CacheOptionsEntry) (client.CacheOptionsEntry, er
 	}
 	return cache, nil
 }
+
+// loadBuildArgEnv loads build-arg values from the environment.
+func loadBuildArgEnv() []string {
+	var ss []string
+	propagatableEnvs := []string{"SOURCE_DATE_EPOCH"}
+	for _, env := range propagatableEnvs {
+		if v, ok := os.LookupEnv(env); ok {
+			bklog.L.Debugf("Propagating %s from the client env to the build arg", env)
+			ss = append(ss, "build-arg:"+env+"="+v)
```

Comment:
It's a tiny thing but I would like it better if we added manual values directly into the map instead of writing a formatted string and then parsing that formatted string in the same code.

Reply:
CLI does not parse `SOURCE_DATE_EPOCH` IIUC.
It is up to the controller and the frontend.

Reply:
That's what the `attrMap` is doing in `ParseOpt`

Reply:
Fixed